### DISCUSSION TITLE
Enhanced table Compatibility to version 7.16.2 of Kibana

### DISCUSSION THREAD

**Discussion by AvinashSugur** [fbaligand/kibana-enhanced-table#250] [Q&A]
HI @fbaligand,

Currently latest version of enhanced table  available is for Version 7.15

Recently Elastic has released 7.16.2 version and we are using enhanced table from 7.9 version and we have upgraded our cluster to 7.16.2 version recently, Can we know when can we expect or available latest enhanced table plugin to install for 7.16.2 version of Kibana.
Which can be compatible for 7.16.2 version.
Let me know asap, we are waiting for latest enhanced table plugin for 7.16.2 compatilibility.

Thank you in Advance.

**Comment by fbaligand** [comment]
I'm very sorry, but I can't give you a date.
I can only say that Kibana 7.16 has a very breaking change for enhanced-table, and so, it's not easy to migrate enhanced table for Kibana 7.16.
That said, it is on the roadmap.
I invite you to follow this issue for that: #237
